Airport_Ticket_Booking_System/Services/CsvService.cs

```diff
@@ -0,0 +1,164 @@
+using Airport_Ticket_Booking_System.Models;
+
+namespace Airport_Ticket_Booking_System.Services;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+public static class CsvService
+{
+    
+
+    public static List<Flight> LoadFlights(string FlightsFilePath)
+    {
+        var flights = new List<Flight>();
```

Comment:
Try enabling Auto-Formatting for C# in your IDE.
It will automatically remove extra spaces, fix indentation, and improve code readability.